zephyr/user_settings.h

```diff
@@ -133,16 +133,42 @@ extern "C" {
     #define NO_SESSION_CACHE /* disable session resumption */
 #endif
 
+/* DTLS */
+#if defined(CONFIG_WOLFSSL_DTLS)
+    #define WOLFSSL_DTLS
+    #define HAVE_SOCKADDR
+#endif
+
 /* PSK */
-#define NO_PSK /* disable pre-shared-key support */
+#if defined(CONFIG_WOLFSSL_PSK)
+    #undef NO_PSK
+    #define WOLFSSL_STATIC_PSK
+#else
+    #define NO_PSK /* disable pre-shared-key support */
+#endif
+
+/* ALPN */
+#if defined(CONFIG_WOLFSSL_ALPN)
+    #define HAVE_ALPN
+#endif
 
+#if defined(CONFIG_WOLFSSL_MAX_FRAGMENT_LEN)
+    #define HAVE_MAX_FRAGMENT
+#endif
+
+#if defined(CONFIG_NET_SOCKETS_SOCKOPT_TLS)
+    #define WOLFSSL_SET_CIPHER_BYTES
```

Comment:
The define here looks to be a left over. Is it still needed?

Reply:
nvm, I see that's used through out

Reply:
No that is for the existing WOLFSSL_SET_CIPHER_BYTES which is needed. Not to be confused with the recently removed WOLFSSL_GET_CIPHER_BYTES